userguide/object_detection/data-preparation.md

```diff
@@ -10,12 +10,23 @@ toolkit, we will rely on many useful SFrame functions.
 The dataset that we will use is [INRIA Annotations for Graz-02
 (IG02)](https://lear.inrialpes.fr/people/marszalek/data/ig02/)[^1][^2]. Our goal is to
 make a detector that can spot bikes and cars. Go ahead and click this link and
-download `ig02-v1.0-bikes.zip` and `ig02-v1.0-cars.zip`. In this tutorial, we
-will assume that you have unzipped these files into `~/Downloads/ig02`. That
-is, you should have the following data structure:
+download `ig02-v1.0-bikes.zip` and `ig02-v1.0-cars.zip`. On unzipping, they will probably 
```

Comment:
Is this true on various operating systems/browsers? I know Safari on macOS behaves this way with multiple zipfile downloads that extract in the same place, but I suspect other OSes behave differently (either by appending a different set of characters, or unpacking into the same folder and clobbering, etc.). If we're going to add this text, we should make sure it works across all browsers on all of the OSes we support. Alternately, let's go with fully terminal-based commands using `curl` that will work consistently.

Reply:
+1 to Zach's comments. Use curl.